packages/apex-parser-ast/src/parser/compilerService.ts

```diff
@@ -122,8 +123,18 @@ export class CompilerService {
     // Create error listener
     const errorListener = new ApexErrorListener(fileName);
 
+    // Determine file type
+    const isTrigger = fileName.endsWith('.trigger');
+    const isAnonymous = fileName.endsWith('.apex') || fileName.endsWith('.anonymous.cls');
+    
+    // For anonymous Apex, wrap content in curly braces since block() expects them
+    let contentToParse = fileContent;
+    if (isAnonymous) {
+      contentToParse = `{\n${fileContent}\n}`;
```

Comment:
Does the additional leading new line cause a line number mismatch from original source when there are "real" problems found in the parsed results?

Reply:
I can test that and get back to you because I'm not actually sure if it's getting included in the line number calc or not.